src/utils/calendar.js

```diff
@@ -79,3 +79,29 @@ export const parseInputToDate = (text, locale) => {
   if (localeDate.isValid()) return localeDate.format('YYYY-MM-DD');
   return '';
 };
+
+export const isDateInRange = (item, min, max) => {
+  const itemDate = moment(item);
+
+  if (min && max) {
+    return itemDate.isBetween(min, max, null, []);
+  }
+  if (min && !max) {
+    return itemDate.isSameOrAfter(min);
+  }
+  if (!min && max) {
+    return itemDate.isSameOrBefore(max);
+  }
+  return true;
+};
+
+export const getExampleDateStrings = () => ({
```

Comment:
This is only used for the Storybook examples and the values are opinionated, but it's nice to keep the examples updated and easy to understand.